src/search1_conversion/convert_result.py

```diff
@@ -167,26 +167,20 @@ def _fetch_narrative_info(results, meta):
     search_params['query'] = {
```

Comment:
Is this still used if the search is dropped?

Reply:
It looks like this variable isn't used any more since the es query is dropped below.

Reply:
This variable isn't used any longer.